content/docs/introduction/technical-preview-free-tier.md

```diff
@@ -29,6 +29,6 @@ Neon stores data in its own internal format. The data size limit in the Free Tie
 
 ## Point-in-time restore
 
-Neon storage consumes extra space in order to support point-in-time restore, which enables you to restore your data to a historical state. Historical data is stored in log-based format.
+Neon storage consumes extra space in order to support point-in-time restore, which enables you to restore your data to a past state. Data history is stored in log-based format.
```

Comment:
'Data history' might be a bit confusing, can we avoid using the word data here? so that distinction between data and history is more clear and not confused even by non-native speakers or people who scan the text quickly.